scripts/configure_libmesh.sh

```diff
@@ -47,6 +47,7 @@ function configure_libmesh()
                --enable-petsc-hypre-required \
                --enable-metaphysicl-required \
                --with-cxx-std-min=2014 \
+               --without-gdb-command \
```

Comment:
```suggestion
               --without-gdb-command \
               --with-future-timpi-dir=${SRC_DIR} \
```

SRC_DIR is always the `libmesh` root directory, regardless of which version of configure we're doing.

Reply:
Like the root source directory? If so, that's not what I want. It needs to be the install directory with `bin,include,lib`

Reply:
You mean the contents of `build`? Then that would be `${SRC_DIR}/build`.

Reply:
Nevermind my deleted comment. I understand now. Need more coffee.